HalloWing_Jump_Scare_Trap/hallowing_jump_scare_trap.py

```diff
@@ -0,0 +1,148 @@
+# HalloWing Jump Scare Trap
+# use PIR sensor, speaker, and servo
+import time
+import array
+import math
+import board
+import displayio
+import pulseio
+from adafruit_motor import servo
+import digitalio
+import touchio
+import audioio
+# Setup LED and PIR pins
+LED_PIN = board.D13  # Pin number for the board's built in LED.
+PIR_PIN = board.SENSE   # Pin port connected to PIR sensor output wire.
+# Setup digital input for PIR sensor:
+pir = digitalio.DigitalInOut(PIR_PIN)
+pir.direction = digitalio.Direction.INPUT
+# Setup digital output for LED:
+led = digitalio.DigitalInOut(LED_PIN)
+led.direction = digitalio.Direction.OUTPUT
+# Setup servo
+# servo = pulseio.PWMOut(board.D4, frequency=50)
+pwm = pulseio.PWMOut(board.D4, duty_cycle=2 ** 15, frequency=50)
+servo = servo.Servo(pwm)
+
+# Setup cap touch button
+ready_button = touchio.TouchIn(board.TOUCH1)
+
+def servo_ready():
+    servo.angle = 0
+def servo_release():
+    servo.angle = 90
+
+# Set servo to ready position
+servo_ready()
+# Function for playing wav file, releasing servo
+def play_wave():
+    wave_file = open("hiss01.wav", "rb")  # open a wav file
+    wave = audioio.WaveFile(wave_file)
+    audio.play(wave)  # play the wave file
+    while audio.playing:  # turn on LED, turn servo
+        led.value = True
+        servo_release()
+        print('Motion detected!')
+    wave_file.close()  # close the wav file
+# Setup audio out pin
+audio = audioio.AudioOut(board.A0)
+
+# tone player setup for status beeps
```

Comment:
replace this with simpleio's tone()?
https://learn.adafruit.com/using-piezo-buzzers-with-circuitpython-arduino/circuitpython#making-tones-with-simpleio-3-19

Reply:
I tried this, but seems like the simplio.tone won't work on A0 because it's looking for PWM. I got this error: `TypeError: extra positional arguments given` Do you have any suggestions? Otherwise I'll leave it as it's currently being done.

Reply:
@kattni can you help @jedgarpark with this? just should just work i thought we added it to simpleio

Reply:
@kattni helped me with this earlier (thanks!) and found that currently the tone code doesn't want to talk to the A0 pin.

Reply:
it should
https://github.com/adafruit/Adafruit_CircuitPython_SimpleIO/blob/master/simpleio.py#L52

Reply:
Looking into this.

Reply:
This is because `tone` hasn't been updated to support the changes to `audiout` introduced in 3.x.

Reply:
oooh @kattni plz put that on your todo list for this week to fix! this code can go live for now until then, then come back and make sure this works

Reply:
Ok! Added to the list. I'll keep @jedgarpark posted.